docs/Trials.md

```diff
@@ -0,0 +1,151 @@
+
+
+## NNI Python Annotation
+
+An alternative to write a trial is to use NNI's syntax for python. Simple as any annotation, NNI annotation is working like comments in your codes. You don't have to make structure or any other big changes to your existing codes. With a few lines of NNI annotation, you will be able to:
+
+* annotate the variables you want to tune 
+* specify in which range you want to tune the variables
+* annotate which variable you want to report as intermediate result to `assessor`
+* annotate which variable you want to report as the final result (e.g. model accuracy) to `tuner`. 
+
+Again, take MNIST as an example, it only requires 2 steps to write a trial with NNI Annotation.
+
+> Step 1 - Update codes with annotations 
+
+The following is a tensorflow code snippet for NNI Annotation, where the highlighted four lines are annotations that help you to: 
+  1. tune batch\_size and dropout\_rate
+  2. report test\_acc every 100 steps
+  3. at last report test\_acc as final result.
+
+What noteworthy is: as these new added codes are annotations, it does not actually change your previous codes logic, you can still run your code as usual in environments without NNI installed.
+
+```diff
+with tf.Session() as sess:
+    sess.run(tf.global_variables_initializer())
++   """@nni.variable(nni.choice(50, 250, 500), name=batch_size)"""
+    batch_size = 128
+    for i in range(10000):
+        batch = mnist.train.next_batch(batch_size)
++       """@nni.variable(nni.choice(1, 5), name=dropout_rate)"""
+        dropout_rate = 0.5
+        mnist_network.train_step.run(feed_dict={mnist_network.images: batch[0],
+                                                mnist_network.labels: batch[1],
+                                                mnist_network.keep_prob: dropout_rate})
+        if i % 100 == 0:
+            test_acc = mnist_network.accuracy.eval(
+                feed_dict={mnist_network.images: mnist.test.images,
+                            mnist_network.labels: mnist.test.labels,
+                            mnist_network.keep_prob: 1.0})
++           """@nni.report_intermediate_result(test_acc)"""
+
+    test_acc = mnist_network.accuracy.eval(
+        feed_dict={mnist_network.images: mnist.test.images,
+                    mnist_network.labels: mnist.test.labels,
+                    mnist_network.keep_prob: 1.0})
++   """@nni.report_final_result(test_acc)"""
+```
+
+**NOTE**: 
+- `@nni.variable` will take effect on its following line, which is an assignment statement whose leftvalue must be specified by the keyword `name` in `@nni.variable`.
+- `@nni.report_intermediate_result`/`@nni.report_final_result` will send the data to assessor/tuner at that line. 
+
+For more information about annotation syntax and its usage, please refer to [Annotation README](../tools/nni_annotation/README.md) . 
+
+
+>Step 2 - Enable NNI Annotation
+
```

Comment:
"In the yaml configure file" -->"In the yml configure file" ??

Reply:
see above